metrics.yaml

```diff
@@ -138,17 +120,25 @@ launch_event:
         type: string
 setting_event:
   current_browser:
-    type: labeled_string
+    type: event
     lifetime: ping
-    send_in_pings:
-      - settings
     description: The external browser has been changed
     bugs:
       - https://linktobugs.page
     data_reviews:
       - https://linktodatareviews.page
     notification_emails:
-      - gsherman@mozilla.com
+      - gsherman@mozilla.co
```

Comment:
Note that you should probably not use your own email address but a mailing-list.

Reply:
I agree. Everything in here is sample code to not throw any errors. notification_emails, bugs, data_reviews are all required fields.

Reply:
The email address is missing a "m".